src/fs/file.rs

```diff
@@ -331,7 +331,7 @@ impl File {
     ///         let (res, buffer) = f.read_exact_at(buffer, 0).await;
     ///         res?;
     ///
-    ///         println!("The bytes: {:?}", buffer);
+    ///         println!("The bytes: {:?}", buffer.into_inner());
```

Comment:
This line 334 println needed changing? If the Slice can't be printed, is that because it doesn't define a debug fmt that pretty prints just the text? What does Vec do, when it's not promised to be holding a sequence of valid UTF-8? I wonder.

It is subtle why this works without qualifying the Vec with the result bytes, at least to me, that into_inner will return the vec but now with a length that is no longer zero. I wonder what will become idiomatic for these returned variables, buffer or slice. I prefer slice but that's just me, I like the name reflecting the type just a little bit when it's convenient, but I know there's just as many people who don't think the name should reflect the type. JTOL.

Reply:
`read_exact_at` only succeeds when it fills the entire buffer, so that `Vec` is going to contain 10 bytes. The `Debug` impl prints them out in the form of array of decimal numbers.